tests/test_simple_range_query.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+from fsql.api import read_partitioned_table
+from fsql.query import ColumnComparator, ColumnRange, SimpleRangeQuery
+
+
+def test_simple_range_query1(tmp_path):
+    # this corresponds to c1=[b, d), c2=*, c3=[c, e)
+    ranges = [
+        ColumnRange("c1", "b", "d"),
+        ColumnRange("c2", "", "", ColumnComparator.wld),
```

Comment:
Does it makes sense to have sth like that? It would work the same way when you exclude this line.

Reply:
I don't like "silent Nones" having meanings -- say you have a function in your script which should produce a ColumnRange for every column. We include the wildcard to not force you to have the return type be Optional[ColumnRange]. It plays a similar role like 0 in integers or identity in functions -- "useless" but makes things complete